config.yaml

```diff
@@ -4,6 +4,8 @@ py_env : scIB-python
 conv_env : scIB-R-conversion
 timing: false
 
+unintegrated_metrics: false
```

Comment:
this is to toggle whether to run unintegrated metrics?

Reply:
yes

Reply:
if not specified, will be interpreted as `False`